zproject/settings.py

```diff
@@ -149,6 +149,8 @@ def get_secret(key):
                     'DBX_APNS_CERT_FILE': None,
                     'EXTRA_INSTALLED_APPS': [],
                     'DEFAULT_NEW_REALM_STREAMS': ["social", "general", "zulip"],
+                    # Used by the generate_realm_creation_link command. Value is in days.
+                    'REALM_CREATION_LINK_VALIDITY': 7,
```

Comment:
would probably make the units be seconds and the value here be `7 * 24 * 3600`, to provide more flexibility (I can imagine someone wanting them to last less than 1 day)